bin/aws-configure

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+#
+# Copyright:: Copyright 2017 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+function usage() {
+  cat /dev/stdin <<DOC
+Usage: ${0##*/} [PROFILE]
+
+A non-interactive version of 'aws configure' that allows you to configure  AWS CLI profiles.
+
+To add a new profile, you MUST specify following environment variables:
+
+  * <PROFILE>_AWS_ACCESS_KEY_ID
+  * <PROFILE>_AWS_SECRET_ACCESS_KEY
+
+You can optionally specify '<PROFILE>_AWS_DEFAULT_REGION' to determine the default region for this profile.
+
+If no PROFILE is specified, ${0##*/} will defer to the value specified in AWS_PROFILE.
+DOC
+}
+
+function enforce_requirements() {
+  # Make sure the AWS CLI is installed
+  which aws >/dev/null 2>1
```

Comment:
You missed an `&`, this should be: (more info [here](http://www.tldp.org/LDP/abs/html/io-redirection.html))
```bash
which aws >/dev/null 2>&1
```